From 79a6bd4a5332cae3ce7e4285773f16e59024d01b Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Thu, 21 May 2020 11:11:16 +0000
Subject: [PATCH] CMakeLists.txt: Don't include CMake/install_config.cmake
 twice

* it duplicates the export for realsense2 and causes cmake to fail with:
-- Configuring done
CMake Error: install(EXPORT "realsense2Targets" ...) includes target "realsense2" more than once in the export set.
CMake Error: install(EXPORT "realsense2Targets" ...) includes target "realsense2" more than once in the export set.
CMake Error: install(EXPORT "realsense2-glTargets" ...) includes target "realsense2-gl" which requires target "realsense2" that is not in this export set, but in multiple other export sets: lib/cmake/realsense2/realsense2Targets.cmake, lib/cmake/realsense2/realsense2Targets.cmake.
An exported target cannot depend upon another target which is exported multiple times. Consider consolidating the exports of the "realsense2" target to a single export.

* it's already included couple lines above where it was moved in
  upstream commit:
  https://github.com/IntelRealSense/librealsense/commit/a54832ab557ff49b24e5840af9e44206a3b655ef

* but the 2nd install_config.cmake include wasn't removed when importing
  this to ros2-gbp/librealsense-release:
  https://github.com/ros2-gbp/librealsense-release/commit/1469d905f04345b4039672dbd501886ba32f5061#diff-af3b638bc2a3e6c650974192a53c7291R130

Upstream-Status: Pending

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6302b9a..3d09964 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,7 +127,6 @@ if(IMPORT_DEPTH_CAM_FW)
 endif()
 
 include(CMake/embedd_udev_rules.cmake)
-include(CMake/install_config.cmake)
 
 if(${ROS_BUILD_TYPE})
     ament_package()
